src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1 calendar__day--1_visible"></div>
+      <div class="calendar__day calendar__day--2 calendar__day--2_visible"></div>
```

Comment:
incorrect implementation
for calendar should be modifiers which will be responsible for start day and month length
no need to add additional classes to calendar__day

```suggestion
    <div class="calendar calendar__month-length--30, calendar__start-day--sun">
      <div class="calendar__day></div>
      <div class="calendar__day></div>
```
![Screenshot from 2023-04-16 17-24-35](https://user-images.githubusercontent.com/33029864/232319763-828789bb-33fa-4b47-9531-f1b21987d6d3.png)